rotkehlchen/tests/unit/accounting/evm/test_paraswap.py

```diff
@@ -0,0 +1,126 @@
+from typing import TYPE_CHECKING
+
+import pytest
+from more_itertools import peekable
+
+from rotkehlchen.accounting.cost_basis.base import CostBasisInfo
+from rotkehlchen.accounting.mixins.event import AccountingEventType
+from rotkehlchen.accounting.pnl import PNL
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.processed_event import ProcessedAccountingEvent
+from rotkehlchen.chain.evm.decoding.paraswap.constants import CPT_PARASWAP
+from rotkehlchen.constants import ONE, ZERO
+from rotkehlchen.constants.assets import A_USDC, A_WBTC
+from rotkehlchen.fval import FVal
+from rotkehlchen.history.events.structures.evm_event import EvmEvent
+from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.tests.utils.accounting import MOCKED_PRICES, TIMESTAMP_1_MS, TIMESTAMP_1_SEC
+from rotkehlchen.tests.utils.factories import make_evm_address, make_evm_tx_hash
+from rotkehlchen.types import Location, Price
+
+if TYPE_CHECKING:
+    from rotkehlchen.accounting.accountant import Accountant
+
+
+@pytest.mark.parametrize('mocked_price_queries', [MOCKED_PRICES])
+@pytest.mark.parametrize('accounting_initialize_parameters', [True])
+def test_paraswap_swap_with_fee(accountant: 'Accountant'):
+    """Test that the fee in paraswap is handled correctly during accounting"""
+    tx_hash = make_evm_tx_hash()
+    user_address = make_evm_address()
+    contract_address = make_evm_address()
+    swap_amount_str = '138.2851'
```

Comment:
you did not do what I asked here: https://github.com/rotki/rotki/pull/7264#discussion_r1450953480

Combine in one line